### PR TITLE
fixed issue Organizations view: 'Sort by' dropdown menu misplaced in Edge browser #2686

### LINKED ISSUE #2686: Organizations view: 'Sort by' dropdown menu misplaced in Edge browser
# Steps  
1.Visit https://staging.apinf.io v 0.46.0  
2.Login as admin 
3.Go to 'Organizations' view  

# Findings  
'Sort by' dropdown menu misplaced in Edge browser     

# Operating system 
Windows 10  

# Browser 
Microsoft Edge v 25.10586.672.0   

# Screenshot  
sortby_menu_misplaced_edge_browser

### PR DESCRIPTION
**Pull request by saransh-dev**
Closes #2686 

## Changes
- add class `pull-left`

## Developer checklist
- [x] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [x] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [x] This pull request does not decrease project test coverage
- [x] If the code changes existing database collection(s), migration has been written
- [x] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @Nazarah @marla-singer 

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/organizations/client/catalog/toolbar/toolbar.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
           <p class="navbar-text sort-filter-toolbar-title">
             {{_ "organizationCatalog_toolbar_sortBy_title" }}
           </p>
-          <div class="form-group sort-options-form">
+          <div class="form-group sort-options-form pull-left">
```
Comment (55) [inline apinf_packages/organizations/client/catalog/toolbar/toolbar.html]: apinf

Extra space in Chrome & Firefox.

Comment (deligent) [reply apinf_packages/organizations/client/catalog/toolbar/toolbar.html]: ok i'll fix it.

File: apinf_packages/organizations/client/catalog/toolbar/toolbar.html [hunk 2]
```diff
@@ -25,21 +25,21 @@
       <!-- Collect the nav links, forms, and other content for toggling -->
       <div class="collapse navbar-collapse sort-filter-toolbar">
         <div class="nav navbar-nav">
-          <p class="navbar-text sort-filter-toolbar-title">
```
Comment (marla-singer) [inline apinf_packages/organizations/client/catalog/toolbar/toolbar.html]: Make sure that other elements don't use CSS class `sort-filter-toolbar-title` and delete related CSS style rule for `.less` file

File: apinf_packages/organizations/client/catalog/toolbar/toolbar.html [hunk 3]
```diff
@@ -25,21 +25,21 @@
...
-            {{_ "organizationCatalog_toolbar_sortBy_title" }}
-          </p>
-          <div class="form-group sort-options-form">
```
Comment (marla-singer) [inline apinf_packages/organizations/client/catalog/toolbar/toolbar.html]: Make sure that other elements don't use CSS class `sort-options-form` and delete related CSS style rule for `.less` file

**Comment by 55** [comment]
@marla-singer let's merge it.

**Pull request merged**